src/OptiHPLCHandler/empower_methodset_method.py

```diff
@@ -87,3 +97,56 @@ def column_temperature(self, temperature: float):
             raise ValueError("No column oven found in method set method.")
```

Comment:
Does this include the column oven in the sample handler?

Reply:
Yes, any instrument method which controls a column oven is included. We need to handle them being turned off.